test/jdk/java/lang/reflect/code/bytecode/TestSmallCorpus.java

```diff
@@ -138,26 +146,48 @@ private void testDoubleRoundtripStability(Path path) throws Exception {
                                         stable++;
                                     }
                                 } catch (Throwable t) {
-                                    error("third lower", t);
+                                    error("third gen", t);
                                 }
                             } catch (Throwable t) {
                                 error("third lift", t);
                             }
                         } catch (Throwable t) {
-                            error("second lower", t);
+                            error("second gen", t);
                         }
                     } catch (Throwable t) {
                         error("second lift", t);
                     }
                 } catch (Throwable t) {
-                    error("first lower", t);
+                    error("first gen", t);
                 }
             } catch (Throwable t) {
                 error("first lift", t);
             }
         }
     }
 
+    private void verify(String category, CoreOp.FuncOp func) {
+        OpWriter.CodeItemNamerOption naming = null;
+        for (Body body : func.bodies()) {
```

Comment:
It's possible to collapse all the loops by traversing the operations, passing in `naming` as the argument e.g.,
```
naming = func.traverse(naming, opVisitor((n, op) -> {
    for (Value v : op.operands()) {
        if (!op.result().isDominatedBy(v)) {
            if (n == null) { n = ... }
            ...
        }
    }
    return n;
}));
```

Reply:
Yes, I'll change it. Original idea was that more verifications will hook also on Body and Block level, however Op-level verification seem to be enough.